### ISSUE TITLE
Creating Functions 3 doesn't let you through

### ISSUE THREAD

**Issue by jared83** [lazerwalker/clojurescript-koans#12]
it is here:
http://clojurescriptkoans.com/#creating-functions/3
doesn't let you through when the gap filled with 
```clojure
multiply-by-5 4
```
i.e.
```clojure
(= 20 (let [multiply-by-5 (partial * 5)] (multiply-by-5 4)))
```
evaluates to true, but the koan doesn't let you through.

edit:
The next one 
http://clojurescriptkoans.com/#creating-functions/4
also doesn't work. This one should actually work without input in the second field.
edit 2:
It didn't work when I simply refreshed the page, but when I did it with [shift + F5] suddenly it all worked. Weird stuff...

**Comment by lazerwalker** [comment]
For your second edit: did hard refreshing fix both issues, or just the second one?

**Comment by lazerwalker** [comment]
The issue in your first one, perhaps, is that the correct answer is 10, not 20. The following works for me:

    (let [multiply-by-five (partial * 5)] (multiply-by-five 2))

**Comment by jared83** [comment]
Hard refreshing fixed both issues (later before I did that I found that I couldn't actually get through any subsequent koans). I reckon that some data got corrupted on its way to me and got cashed that wayin my browser (newest ver. of chrome). 
It is definitely 20. The koan is hard-wired to check for equality to 20.
I think much bigger issue is with the Firefox I issued just before this one.
When I go directly to 
http://clojurescriptkoans.com
it actually renders the page just right, but when I click the "Let us begin" button, the URL changes but nothing happens.

**Comment by lazerwalker** [comment]
"The koan is hard-wired to check for equality to 20" -> can you specify exactly which koan you're talking about? Your initial post made me think you were talking about #3 on functions, which doesn't seem to be the case. Want to make sure we're both on the same page, and eliminate the possibility there's some bug with the wrong koan loading: https://github.com/lazerwalker/clojurescript-koans/blob/master/src/koans/meditations/06_functions.cljs

**Comment by lazerwalker** [comment]
Ah, hah. Creating-functions vs functions. Totally a reading comprehension failure on my part.

**Comment by jared83** [comment]
Happens to all of us :)
BTW, thank you for this enlightening service. I love the form. It would be cool if you could keep adding new koans as you yourself are learning new things about this marvellous technology. I'm not talking about anything on the shape of 4clojure (though it is very cool as well), but rather a way to get people familiar with additional more functions from cljs.core, core.async, how to make macros, core.nodejs etc.
Also maybe some db so that the experience could be more personalised with user accounts where one could see their path to enlightenment etc. Firebase could be of help here. This way you could avoid any changes to your backend. And then there could be a footer for each koan with some additional resources regarding the subject of the koan at hand...
Just a couple of suggestions :)

**Comment by lazerwalker** [comment]
Thanks for the kind words! Glad you enjoy it :)

I'm afraid I don't have too much time to work on this myself any more, and since none of my work these days is in Clojure or Clojurescript it's hard to justify dedicating what little side-project time I have to making large-scale improvements, as lovely as they'd be.

That being said, if you're interested in helping out with any of this and contributing, it would be completely welcome and appreciated!

**Comment by jared83** [comment]
- " and since none of my work these days is in Clojure or Clojurescript" - uhh... how unfortunate, no? Such beautiful and versatile language and so underused. Is it difficult to find job as clojure developer? When I first learned about lisp (in the form of clojure) it resonated with me very deeply. Now that I completed the koans and watched quite a few videos about clojure & clojurescript I feel like I don't want to ever touch imperative. Well, maybe with the exception of Angular 2 + rxjs + react.js for rendering. The newest version of Angular simply offers too many perks to ignore with a straight face. Wish someone would port it to clojurescript though. 
I wish I could contribute, but I am afraid I still need to learn a whole lot before I'll be able to be of use to the open source community at large.
Once I feel more confident I might drop by and thank you for your work on this by contributing.
cheers
btw. I noticed recursion koans commented out. Why? Was it causing web browsers to crash / stack overflow? I looked into recursion issue in clojurescript and it seems to have made some strides recently.
